t/redis.t

```diff
@@ -0,0 +1,37 @@
+use strict;
+use warnings;
+
+use Date::Utility;
+use Test::More;
+use Test::Exception;
+use Data::Chronicle::Writer;
+require Test::NoWarnings;
+use RedisDB;
+
+my $data = {sample => 'data'};
+
+subtest "Call Set after dropping the connection" => sub {
```

Comment:
There should be a way to skip this test (`plan 'skip_all'`) especially on CPAN testers matrix since those won't have a Redis or Postgres server.